proto/spire/plugin/server/upstreamauthority/v1/upstreamauthority.proto

```diff
@@ -57,10 +27,36 @@ service UpstreamAuthority {
     // Implementation note:
     // The stream should be kept open in the face of transient errors
     // encountered while tracking changes to the upstream JWT keys as SPIRE
-    // core will not reopen a closed stream until the next JWT key rotation.
+    // server will not reopen a closed stream until the next JWT key rotation.
     rpc PublishJWTKey(PublishJWTKeyRequest) returns (stream PublishJWTKeyResponse);
+}
+
+message MintX509CARequest {
+    // Required. Certificate signing request (PKCS#10)
+    bytes csr = 1;
+
+    // Optional. Preferred TTL is the TTL preferred by SPIRE Server for signed CA. If
+    // zero, the plugin should determine its own TTL value. Plugins are free to
+    // ignore this and use their own policies around TTLs.
+    int32 preferred_ttl = 2;
+}
 
-    // Standard SPIRE plugin RPCs
-    rpc Configure(spire.common.plugin.ConfigureRequest) returns (spire.common.plugin.ConfigureResponse);
-    rpc GetPluginInfo(spire.common.plugin.GetPluginInfoRequest) returns (spire.common.plugin.GetPluginInfoResponse);
+message MintX509CAResponse {
+    // Required on the first response. Contains ASN.1 encoded certificates
+    // representing the X.509 CA along with any intermediates necessary to
+    // chain back to a certificate present in the upstream_x509_roots.
```

Comment:
Some more detail here will be useful for plugin authors, especially those that may be only passingly familiar with x509

Specifically, text on the ordering requirements would help a lot I think